### PR TITLE
updated to libmdbx v0.9.1

### PR DESCRIPTION
**Pull request by kayyagari**
o updated to libmdbx v0.9.1
o bumped the version number to 0.7.2

### CODE REVIEW

**Comment by vorot93** [comment]
I think this PR is already stale as the latest version is 0.9.2.

File: mdbx-sys/Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "mdbx-sys"
 # NB: When modifying, also modify html_root_url in lib.rs
 # NB: The version must match the version of mdbx
-version = "0.7.1"
+version = "0.7.2"
```
Comment (Kerollmops) [inline mdbx-sys/Cargo.toml:5]: Hey thank you, could you please update the `lib.rs` file too?

Comment (Kerollmops) [inline mdbx-sys/Cargo.toml:5]: Are you sure this is not breaking and does this version follow the libmdbx one?

**Review by Kerollmops — CHANGES_REQUESTED** [review]
Thank you very much for your work, and sorry for the delay!

**Pull request closed without merging**